scss/_base_fontfaces.scss

```diff
@@ -1,187 +1,80 @@
-        }
-      } @else {
-        @font-face {
-          font-display: $font-display-option;
-          font-family: 'Ubuntu';
-          font-style: normal;
-          font-weight: $font-weight-regular-text;
-          src: url('#{$assets-path}e8c07df6-Ubuntu-L_W.woff2') format('woff2'), url('#{$assets-path}8619add2-Ubuntu-L_W.woff') format('woff');
-        }
-
-        @font-face {
-          font-display: $font-display-option;
-          font-family: 'Ubuntu';
-          font-style: normal;
-          font-weight: $font-weight-bold;
-          src: url('#{$assets-path}fff37993-Ubuntu-R_W.woff2') format('woff2'), url('#{$assets-path}7af50859-Ubuntu-R_W.woff') format('woff');
-        }
-
-        @font-face {
-          font-display: $font-display-option;
-          font-family: 'Ubuntu';
-          font-style: italic;
-          font-weight: $font-weight-regular-text;
-          src: url('#{$assets-path}f8097dea-Ubuntu-LI_W.woff2') format('woff2'), url('#{$assets-path}8be89d02-Ubuntu-LI_W.woff') format('woff');
-        }
-
-        @font-face {
-          font-display: $font-display-option;
-          font-family: 'Ubuntu';
-          font-style: italic;
-          font-weight: $font-weight-bold;
-          src: url('#{$assets-path}fca66073-ubuntu-ri-webfont.woff2') format('woff2'), url('#{$assets-path}f0898c72-ubuntu-ri-webfont.woff') format('woff');
-        }
-
-        @font-face {
-          font-display: $font-display-option;
-          font-family: 'Ubuntu';
-          font-style: normal;
-          font-weight: $font-weight-display-heading;
-          src: url('#{$assets-path}7f100985-Ubuntu-Th_W.woff2') format('woff2'), url('#{$assets-path}502cc3a1-Ubuntu-Th_W.woff') format('woff');
-        }
+      // Default font-display option
+      @font-face {
+        font-family: 'Ubuntu';
+        font-stretch: 100%; /*  min and max value for the width axis, expressed as percentage */
+        font-style: normal;
+        font-weight: 100 800; /*  min and max value for the weight axis */
+        src: url('#{$assets-path}f3b9cc97-Ubuntu[wdth,wght]-latin (1).woff2') format('woff2-variations');
+      }
 
-        @font-face {
-          font-display: $font-display-option;
-          font-family: 'Ubuntu Mono';
-          font-style: normal;
-          font-weight: $font-weight-regular-text;
-          src: url('#{$assets-path}fdd692b9-UbuntuMono-R_W.woff2') format('woff2'), url('#{$assets-path}85edb898-UbuntuMono-R_W.woff') format('woff');
-        }
+      @font-face {
+        font-family: 'Ubuntu';
+        font-stretch: 100%; /*  min and max value for the width axis, expressed as percentage */
+        font-style: italic;
+        font-weight: 100 800; /*  min and max value for the weight axis */
+        src: url('#{$assets-path}c1b12cdf-Ubuntu-Italic[wdth,wght]-latin (1).woff2') format('woff2-variations');
```

Comment:
Could we avoid having unnecessary stuff in font URLs?

Having spaces and `(1)` caused by downloading same name multiple times looks messy.
I know that right now it means uploading it again probably, unless someone from backend guys can help us renaming those on asset server.

Reply:
It's important these URLs look clean and clear.

Reply:
Oh my bad I didn't think it mattered. Do we want to agree on a format before I reupload them?